configs/xfce/CNCHI_UPDATER.pot

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
```

Comment:
@lots0logs I was getting a few errors from symlinks when I was running the build script, I'm going to assume I did something, and ended up just replacing them all to get the issue out of my hair.  I'll go through em all and replace them back.